src/Apps/Home/Components/HomeContentCards.tsx

```diff
@@ -121,10 +97,18 @@ export const HomeContentCards: React.FC = () => {
   ]
 
   const realCards = cards.map((card, index) => (
-    <ContentCard card={card} key={card.id} index={index} />
+    <HomeContentCard card={card} key={card.id} index={index} />
   ))
 
   const heroCards = cards.length < 1 ? placeholderCards : realCards
 
-  return <HeroCarousel>{heroCards}</HeroCarousel>
+  const handleChange = newIndex => {
```

Comment:
What about this index is "new"?

Reply:
Yeah idk, I guess I was thinking "new" in terms of it changing from the previous value but that's silly - will update!